src/main/resources/google_checks.xml

```diff
@@ -361,6 +355,12 @@
       <message key="name.invalidPattern"
              value="Method name ''{0}'' must match pattern ''{1}''."/>
     </module>
+    <module name="SuppressionXpathSingleFilter">
+      <property name="checks" value="MethodName"/>
+      <property name="query" value="//METHOD_DEF[
+                                     .//ANNOTATION/IDENT[@text='Test']]//*"/>
+      <property name="message" value="'[a-z][a-z0-9][_a-zA-Z0-9]*'.*"/>
+    </module>
```

Comment:
Moved the suppression next to its module for easy reference, should have done this at: https://github.com/checkstyle/checkstyle/pull/15515 but it slip through my mind so had to do it here